a10_octavia/controller/worker/tasks/a10_database_tasks.py

```diff
@@ -75,6 +75,10 @@ def execute(self, amphora, loadbalancer, role):
             'DEFAULT', 'DEFAULT_VTHUNDER_PASSWORD').replace('"', '')
         axapi_version = int(
             self.config.get('DEFAULT', 'DEFAULT_AXAPI_VERSION').replace('"', ''))
+        port = int(
+            self.config.get('DEFAULT', 'DEFAULT_VTHUNDER_PORT').replace('"', ''))
+        protocol = self.config.get(
+            'DEFAULT', 'DEFAULT_VTHUNDER_PROTOCOL').replace('"', '')
```

Comment:
FUTURE REF: To be taken care after Afrin's PR config PR merge, putting just for future reference